main.go

```diff
@@ -82,7 +82,20 @@ func main() {
 		}
 	}
 
-	funcs, err := pwru.GetFuncs(flags.FilterFunc, btfSpec, flags.KMods)
+	var useKprobeMulti bool
+	if flags.Backend != "" && (flags.Backend != "kprobe" && flags.Backend != "kprobe-multi") {
```

Comment:
Nit: maybe move the `"kprobe"` and `"kprobe-multi"` strings into consts, so they could be reused below and in the usage message for `--backend`?